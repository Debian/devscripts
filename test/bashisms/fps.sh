#!/bin/sh

echo -e "BASHISM" \
    "something else \n"

#exec ${loclibdir}/tkcon.tcl \
#        -eval "source ${loclibdir}/console.tcl" \
#        -slave "package require Tk; set argc $#; set argv [list $*]; \
#        source ${loclibdir}/xcircuit.tcl"
