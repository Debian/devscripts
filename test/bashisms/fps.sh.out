possible bashism in bashisms/fps.sh line 4 (echo -e):
echo -e "BASHISM" \
    "something else \n"
